operations/firmware/Firmware_Upgrade_using_SPP_on_HPE_ProLiant_Servers.md

```diff
@@ -0,0 +1,89 @@
+* Go to iLO Repository, you should be able to see the uploaded component. Click on `Install Component` and then Click on `TPM Override` then click on `Add to the Installation queue`
+* You can check for the status message in Installation Queue. If you see any failed instance in Queue, then delete that instance else installation of next component would not get start if expiration is set to never for the failed instance.
+
+## Deploy SPP ISO image as a Whole
+
+### Pre-requisites
+
+* Downloaded SPP ISO image
+* ILO accessible on system where you are flashing firmware using SPP ISO image
+* Web Browser
+
+The SPP ISO Image can be deployed locally or remotely.
+
+## Deploying the ISO Image using the local method
+
+* Login to ILO in web browser
+* Select `Remote console & Media` on to the left side
+* Launch the remote console either `HTML5` or `.NET` console or through `HPLOCONS`. Best option would be .NET and HPLOCONS
+* Mount the ISO image either local location of the image or URL path. If URL path is given make sure that is in same network as ILO network
+* Do cold reboot or Reset the node from iLO remote console
+* Press `F9` and go to `One time Boot` menu and select the mounted image, if URL is given then you see the URL path.
+* Select `Interactive` option
+* Accept and proceed
+* Click on `Firmware Update` and then Click OK
+* Inventory takes some time, wait until it completes and then click Next
+* Select the components that needs firmware flashing, you can also check for the version by clicking view details for each component.
+If its already up to date then by default it will not be selected, you can force update the component. Click on TPM override and then deploy
+* After deployment, check for the status and reboot the node
+
+## Deploying the ISO Image using the remote method
+
+* Mount the ISO image or copy and extract the ISO image.
+* If on a Windows Server then click on `launch_sum.bat` file, if on a Linux then `launch_sum.sh` file
+* Continue in the browser window that was opened
+* If `localhost Guided Update` is selected, then it updates the components on the server from where `launch_sum` execution file is launched.
+* You can deploy on one or more than one target nodes by selecting the option Nodes
+* Click on `Nodes`, then `Add Node`. Give the ILO IP in IPV4 section (You can also give range of nodes), select Node type as iLO and give ILO credentials by scrolling down
+* Once the Discovery is done click on `Add Inventory`, once inventory is completed click on Next
+* Inventory takes some time, wait until it completes and then click Next
+* Select the components that needs firmware flashing, you can also check for the version by clicking view details for each component. If its already up to date then by default it will not be selected, you can force
+  update the component. Click on `TPM Override` and then `deploy`
+* After deployment, check for the status and reboot the node
+* Repeat this for all the nodes added.

```

Comment:
```suggestion
1. Mount the ISO image or copy and extract the ISO image.
1. If on a Windows Server then click on `launch_sum.bat` file, if on a Linux then `launch_sum.sh` file.
1. Continue in the browser window that was opened.
1. If `localhost Guided Update` is selected, then it updates the components on the server from where `launch_sum` execution file is launched.
1. You can deploy on one or more than one target nodes by selecting the option Nodes.
1. Click on `Nodes`, then `Add Node`. Give the ILO IP in IPV4 section (You can also give range of nodes), select Node type as iLO and give ILO credentials by scrolling down.
1. Once the Discovery is done, click on `Add Inventory`. Once inventory is completed, click on `Next`.
1. Inventory takes some time, wait until it completes and then click `Next`.
1. Select the components that needs firmware flashing, you can also check for the version by clicking view details for each component. If it is already up to date then by default it will not be selected, you can force
  update the component. Click on `TPM Override`, and then `deploy`.
1. After deployment, check for the status and reboot the node
1. Repeat this for all the nodes added.

```